dcap-ql/src/quote.rs

```diff
@@ -191,13 +248,27 @@ impl<'a> Quote3Signature<'a> for Quote3SignatureEcdsaP256<'a> {
             qe3_signature,
             authentication_data,
             certification_data_type,
-            certification_data: data.into(),
+            certification_data: data,
         })
     }
+
+    /// Verfies the signature over the quote header and ISV enclave report by the attestation key
+    #[cfg(feature = "verify")]
+    fn verify(&'a self, quote: &[u8]) -> Result<&'a Self> {
```

Comment:
I think it's good to keep this as its own self-contained function. But it should be named `verify_quote_signature` and be on `impl Quote3SignatureEcdsaP256`